### DISCUSSION TITLE
Try/Catch middleware_group throwable exception

### DISCUSSION THREAD

**Discussion by chaudigv** [livewire/livewire#1916] [1. Help]
In `livewire/config.php`, I added `'middleware_group'  => ['web', 'throttle:2,1'],`. This means that upon the 3rd request I should expect **429 | Too Many Requests**, which I do.

However, I would like to catch **Illuminate\Http\Exceptions\ThrottleRequestsException** exception, but I am unsure of how or where exactly to handle that. I read the [doc](https://laravel-livewire.com/docs/2.x/authorization) but couldn't find more info.

Please advice.

**Comment by joshhanley** [comment]
You will need to have a look at Laravel information to work this out. 

Have a look the link below. I think he has complicated it a bit, I would personally, just extend the base `ThrottleRequests` class like is suggested at the end of the article, and overload `buildResponse` method.

https://thedevsaddam.gitbooks.io/off-time-story/content/how_to_customize_laravel_throttle_message_response.html

There is another way, but modifying the HttpException class, check out some of the responses at the end of this thread.
https://github.com/laravel/framework/issues/13623
